guides/md/serialization_and_saving.md

```diff
@@ -292,17 +291,27 @@ reconstructed_model.fit(test_input, test_target)
 
 <div class="k-default-codeblock">
 ```
-4/4 [==============================] - 0s 967us/step - loss: 0.8106
-4/4 [==============================] - 0s 1ms/step - loss: 0.7184
+4/4 [==============================] - 0s 2ms/step - loss: 0.2321
+4/4 [==============================] - 0s 1ms/step - loss: 0.2229
 
-<tensorflow.python.keras.callbacks.History at 0x151d5ac90>
+<keras.callbacks.History at 0x7894c828b470>
 
 ```
 </div>
-#### Limitations
+### Format Limitations
+
+SavedModel limitations:
+
+- Can be slower and bulkier than H5, since it saves the traced TF graphs of each layer
+- Can not serialize the mask argument in custom layers (`layer.supports_masking=True`)
```

Comment:
Rather than `layer.supports_masking=True` this refers to `layer(..., mask=mask_value)`

Reply:
Changed